docs/appendix/glossary/process.md

```diff
@@ -11,3 +11,5 @@ Processes are binaries or scripts that run on a {{< glossary_tooltip term_id="pa
 You can use processes to create a new local instance of `viam-server` to implement drivers for custom {{< glossary_tooltip term_id="component" text="components" >}}, or to run a client application, for example.
 They provide an OS-specific process managed by `viam-server` to either run once or indefinitely.
 For example, you could use a process to run a camera server.
+
+For information on how to configure a process, see [Configure a Robot](/manage/configuration/#processes).
```

Comment:
Can you add the link to line 4

Reply:
This one was actually intentional; since it's not about what processes are in general, but more specific to configuring them, I thought it'd be better if folks land here instead of there, and then can click this if they want.

Reply:
ok